CHANGELOG.md

```diff
@@ -4,6 +4,61 @@
 `psql` with the `-X` flag to prevent any `.psqlrc` commands from
 accidentally triggering the load of a previous DB version.**
 
+## 2.12.0 (2023-09-20)
+
+This release contains bug fixes since the 2.11.2 release.
```

Comment:
Initial suggestion on mentioning performance improvements for compression and caggs

```suggestion
This release contains performance improvements for compressed hypertables and continuous aggregates
and bug fixes since the 2.11.2 release.
```

@svenklemm @fabriziomello @antekresic do you think that we could elevate this by adding a quick list of noteworthy features or should we leave this to that?

Reply:
Leave it as is IMHO